tests/conftest.py

```diff
@@ -59,11 +62,3 @@ def mock_firebase_send(mocker: MockerFixture, firebase_message_id_send):
     mock = mocker.patch("fcm_django.models.messaging.send")
     mock.return_value = firebase_message_id_send
     return mock
-
-
-@pytest.fixture
-def mock_fcm_device_deactivate(mocker: MockerFixture):
-    return mocker.patch(
-        "fcm_django.models.FCMDevice.deactivate_devices_with_error_result",
```

Comment:
Since the model could be Swapped, this fixture may not work. So I removed it and corrected a few tests.